integration/src/main/java/scripts/testscripts/DatasetOperations.java

```diff
@@ -161,12 +164,26 @@ private void previewUserJourney(StorageSystem storageSystem, String sourceId)
     datasetId = null;
   }
 
+  private ObjectNode expectedMetadataResponse(ObjectNode metadata, StorageSystem storageSystem) {
+    ObjectNode expectedMetadata = metadata.deepCopy().put("id", datasetId.toString());
+    if (storageSystem.equals(StorageSystem.TDR)) {
+      expectedMetadata
+          .put("phsId", "1234")
```

Comment:
I don't think `phsId` should be exposed as a property, unless there's a specific need in the UI for this attribute.

Reply:
It sounds like there definitely is a UI need!